### PR TITLE
add brandon's info for rnn lecture

### PR DESCRIPTION
**Pull request by brandons209**

### CODE REVIEW

File: sp19/syllabus.yml [hunk 1]
```diff
@@ -46,9 +46,15 @@ primary:
 - name: ""
   file: "recurrent-nets"
   covr: ""
-  tags: ["neural-nets", "recurrent-nets", ]
-  inst: ["ionlights"]
+  tags: ["neural-nets", "recurrent-nets", "LSTMs", "Embeddings"]
+  inst: ["ionlights", "brandons209"]
```
Comment (ionlights) [inline sp19/syllabus.yml]: @brandons209 prepend your name, i'm essentially an editor on this, even though i'm anticipating major usage of my source material. 😅

**Review by jmuchovej — CHANGES_REQUESTED** [review]
@brandons209 prepend your name in the `inst` list – @ionlights is essentially an editor, even though you'll likely draw from most of his source material. 😅

**Review by brandons209 — COMMENTED** [review]
my name is already there.

**Review by brandons209 — COMMENTED** [review]
okay fixed

**Review by jmuchovej — APPROVED** [review]

**Pull request merged**